src/components/themes/appletv/theme.css

```diff
@@ -462,3 +462,9 @@ html {
 .metadataSidebarIcon {
     color: #00a4dc
 }
+
+.mediaInfoContent {
```

Comment:
I would remove this from the theme styles and add it to `site.css` instead.

Reply:
done